src/genie/libs/parser/iosxr/show_isis.py

```diff
@@ -5134,3 +5136,99 @@ def get_value(value):
                 continue
 
         return result_dict
+
+################################################################################
+
+"""Schema for 'show isis segment-routing srv6 locators'"""
+class ShowIsisSegmentRoutingSrv6LocatorsSchema(MetaParser):
```

Comment:
Please add a docstring for this schema
```suggestion
class ShowIsisSegmentRoutingSrv6LocatorsSchema(MetaParser):
    """Schema for:
    * show isis segment-routing srv6 locators
    * show isis instance {instance} segment-routing srv6 locators
    """
```

Reply:
Updated in latest commit